fix: catch errors and inform user of the need to register voting rights [LW-11518]

DominikGuzei opened the pull request:
Properly handle the case when a Conway era wallet that has staking rewards but did not participate in governance (register voting rights) yet can't send transactions due to blocked staking rewards.

**Please note:** This PR is based on https://github.com/input-output-hk/daedalus/pull/3229 to work with the latest cardano-node and cardano-wallet which needs to be merged first.

## Screenshots

![image](https://github.com/user-attachments/assets/2b4054d3-b1ff-4c2e-8ef2-a58ab348fc8d)

![image](https://github.com/user-attachments/assets/94419245-de16-4e72-a098-43ef18276b04)

![image](https://github.com/user-attachments/assets/7bc206a9-6113-478e-89df-e1d598077854)

## Todos

- [x] Catch the error from the node and show some message in the send tx confirm dialog
- [x] Disable the send button in the confirmation dialog if this error happened
- [x] Link to the governance tab from the error link

DominikGuzei commented on source/renderer/app/stores/WalletsStore.ts:
```diff
@@ -837,25 +837,29 @@ export default class WalletsStore extends Store {
-    this.actions.dialogs.closeActiveDialog.trigger();
-    this.sendMoneyRequest.reset();
-    this.goToWalletRoute(wallet.id);
+    try {
```
Wrapping this in a `try` catch block was necessary because otherwise it crashed Daedalus … im not sure how it worked with other types of errors though 🤔

przemyslaw-wlodek commented on source/renderer/app/i18n/locales/ja-JP.json:
```diff
@@ -34,6 +34,7 @@
   "api.errors.WalletAlreadyImportedError": "インポートしようとしているウォレットは既に存在します。",
   "api.errors.WalletAlreadyRestoredError": "復元しようとしているウォレットは既に存在します。",
   "api.errors.WalletFileImportError": "ウォレットをインポートできませんでした。有効なファイルを指定していることを確認してください。",
+  "api.errors.conwayWalletNotDelegatedToDRep": "!!!conwayWalletNotDelegatedToDRep",
```
Note: This will be delivered in the next PR.

przemyslaw-wlodek reviewed (approved):
Great job @DominikGuzei ! 🚀

DominikGuzei commented:
This PR needs to wait for https://github.com/input-output-hk/daedalus/pull/3229 to be merged (base branch)

DominikGuzei commented:
@szymonmaslowski @szymonmaslowski I rebased this PR on `feat/LW-11519-governance-voting-tab` and added the final translations (Japanese) so that we have everything in one place. I tested that the error link correctly redirects to the new governance tab. Feel free to merge after review 🙏

szymonmaslowski reviewed (approved):
GJ!
